docs/Gemfile

```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
+gem 'github-pages', group: :jekyll_plugins
```

Comment:
This is just for local preview?

Reply:
Correct. Seems useful. I'll document it.